elements/lisk-tree/src/merkle_tree.ts

```diff
@@ -83,12 +176,159 @@ export class MerkleTree {
 	}
 
 	public clear(): void {
-		this._dataLength = 0;
-		this._data = {};
+		this._width = 0;
+		this._hashToBuffer = {};
```

Comment:
Even for an empty tree, I would insert the emptyHash in `_hashToBuffer`, for consistency